zubhub_frontend/zubhub/src/api/api.js

```diff
@@ -1,5 +1,5 @@
 import i18next from 'i18next';
-
+let jsonActivities = require('./genericActivities.json');
```

Comment:
This needs to be changed to import from require

Reply:
And I think the activities should come from api otherwise every time user opens the page in new tab or reload the page activities are lost.

@tuxology

Reply:
> And I think the activities should come from api otherwise every time user opens the page in new tab or reload the page activities are lost.
> 
> @tuxology

Hi @kamthamc , Thanks for taking the time to review.
I'm using this json file just temporarily until creating the backend services for the new activities feature